staging/src/k8s.io/apimachinery/pkg/api/validate/schema.go

```diff
@@ -63,12 +63,10 @@ func Forbidden[T comparable](_ operation.Context, fldPath *field.Path, value, _
 // Optional verifies that the specified value is not the zero-value for its
 // type. This is identical to Required, but the caller should treat an error
 // here as an indication that the optional value was not specified.
-func Optional[T comparable](_ operation.Context, fldPath *field.Path, value, _ *T) field.ErrorList {
```

Comment:
Update the above `Forbidden()` to `any` as well?

Reply:
Updated.